src/support/cleanse.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_CLEANSE_H
```

Comment:
This should be ``BITCOIN_SUPPORT_CLEANSE_H`` and you are currently missing the header end comment :).